src/order/atoms.lean

```diff
@@ -50,6 +50,22 @@ or.imp (ha.2 b) eq_comm.2 (lt_or_eq_of_le hab)
 
 end is_coatom
 
+section pairwise
+
+lemma inf_eq_bot_of_atom_ne_atom [semilattice_inf_bot α] {a b : α}
```

Comment:
Maybe `is_atom.inf_eq_bot_of_ne`?
Whatever you call it, add another version called `disjoint_of_atom_ne_atom` or `is_atom.disjoint_of_ne`, with the result `disjoint a b`. They should be defeq, so you won't need to copy the proof.